### ISSUE TITLE
Incorrect typings in 5.0.0

### ISSUE THREAD

**Issue by Enngage** [Agranom/ngx-material-timepicker#195]
Hey,

just upgraded to `5.0.0` and found out there are incorrect typings preventing my app from building in production mode.

```
ERROR in node_modules/ngx-material-timepicker/ngx-material-timepicker.d.ts.ɵt.html(5,12): Argument of type 'number | null' is not assignable to parameter of type 'string | number'.
  Type 'null' is not assignable to type 'string | number'.
```

**Comment by Agranom** [comment]
Hey, I'll check

**Comment by Agranom** [comment]
It should be fixed in v5.1.0
